### PR TITLE
fix(billing.mean): change message for payment mean addition

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

Change success message to info message and reword

MBE-182

### CODE REVIEW

File: client/app/account/billing/mean/add/billing-mean-add.controller.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * @name Billing.controllers.Method.Add'
  * @description
  */
-angular.module('Billing.controllers').controller('Billing.controllers.Mean.Add', ($scope, $location, $q, $translate, IBAN_BIC_RULES, BillingMean, BillingUser, Alerter, User) => {
+angular.module('Billing.controllers').controller('Billing.controllers.Mean.Add', ($location, $q, $scope, $translate, IBAN_BIC_RULES, Alerter, BillingMean, BillingUser, paymentMethodHelper, User) => {
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply client/app/account/billing/mean/add/billing-mean-add.controller.js]: Also, constants are supposed to be last

File: client/app/account/billing/mean/add/billing-mean-add.controller.js [hunk 2]
```diff
@@ -130,20 +130,20 @@ angular.module('Billing.controllers').controller('Billing.controllers.Mean.Add',
 
     return BillingMean.post(meanData)
       .then((response) => {
-        $scope.mean = {};
+        const url = _.get(response, 'url');
```
Comment (FredericEspiau) [inline client/app/account/billing/mean/add/billing-mean-add.controller.js]: could `then({ url} =>` work ?

File: client/app/account/billing/mean/add/billing-mean-add.html [hunk 3]
```diff
@@ -232,24 +232,27 @@
...
-                        data-ng-show="canPaymentTypeSetDefaultAtCreation(mean.type)">
+                     data-ng-if="canPaymentTypeSetDefaultAtCreation(mean.type) && hasDefaultPaymentMean">
                     <label>
                         <input type="checkbox"
                                 data-ng-model="mean.setDefault" />
                         <span data-translate="payment_mean_set_default_after_validation"></span>
                     </label>
                 </div>
 
+                <p><strong data-ng-if="!hasDefaultPaymentMean" data-translate="payment_mean_set_as_default"></strong></p>
```
Comment (FredericEspiau) [inline client/app/account/billing/mean/add/billing-mean-add.html]:
```suggestion
               <strong class="d-block" data-ng-if="!hasDefaultPaymentMean" data-translate="payment_mean_set_as_default"></strong>
```

File: client/app/account/billing/translations/Messages_fr_FR.xml [hunk 4]
```diff
@@ -813,7 +814,8 @@
    <translation id="payement_mean_add_creditCard" qtlid="177288">Générer un bon de commande</translation>
    <translation id="payment_mean_get_validation" qtlid="177301">Récupérer la procédure de validation</translation>
    <translation id="payment_mean_return_by_mail_procedure" qtlid="325588">Procédure à retourner par voie postale</translation>
-   <translation id="paymentType_add_success_url" qtlid="363878"><a href="{0}" target="_blank">Cliquez ici</a> pour finaliser l'enregistrement de votre moyen de paiement.</translation>
+   <translation id="paymentType_add_validation_description">Vous devez valider l'ajout de votre moyen de paiement.</translation>
+   <translation id="paymentType_add_success_url"><a href="{0}" target="_blank">Cliquez ici</a> pour être redirigé vers une page sécurisée et finaliser l'opération.</translation>
```
Comment (FredericEspiau) [inline client/app/account/billing/translations/Messages_fr_FR.xml]: Possible to extract the link ? Also to add `ref="noopener "` ?

File: client/app/account/billing/billingApp.module.js [hunk 5]
```diff
@@ -133,7 +133,7 @@ angular
         $stateProvider.state('app.account.billing.payment.mean.add', {
           url: '/add',
           templateUrl: `${BILLING_BASE_URL}mean/add/billing-mean-add.html`,
-          controller: 'Billing.controllers.Mean.Add',
+          controller: 'billingMeanAddCtrl',
```
Comment (antleblanc) [inline client/app/account/billing/billingApp.module.js]: According to the state name `app.account.billing.payment.mean.add` why not naming your controller as `billingPaymentMeanAddCtrl`?

File: client/app/account/billing/translations/Messages_fr_FR.xml [hunk 6]
```diff
@@ -813,7 +814,8 @@
    <translation id="payement_mean_add_creditCard" qtlid="177288">Générer un bon de commande</translation>
    <translation id="payment_mean_get_validation" qtlid="177301">Récupérer la procédure de validation</translation>
    <translation id="payment_mean_return_by_mail_procedure" qtlid="325588">Procédure à retourner par voie postale</translation>
-   <translation id="paymentType_add_success_url" qtlid="363878"><a href="{0}" target="_blank">Cliquez ici</a> pour finaliser l'enregistrement de votre moyen de paiement.</translation>
+   <translation id="paymentType_add_validation_description">Vous devez valider l'ajout de votre moyen de paiement.</translation>
+   <translation id="paymentType_add_success_url"><a href="{0}" target="_blank" rel="noopener">Cliquez ici</a> pour être redirigé vers une page sécurisée et finaliser l'opération.</translation>
```
Comment (FredericEspiau) [inline client/app/account/billing/translations/Messages_fr_FR.xml]: Extraction ? :D

Comment (marie-j) [reply client/app/account/billing/translations/Messages_fr_FR.xml]: Unfortunately, I didn't find any cleaner solution

**Pull request merged**